pkg/charts/cni/values.go

```diff
@@ -17,22 +17,28 @@ const (
 
 // Values contains the top-level elements in the cni Helm chart
 type Values struct {
-	InboundProxyPort    uint   `json:"inboundProxyPort"`
-	OutboundProxyPort   uint   `json:"outboundProxyPort"`
-	IgnoreInboundPorts  string `json:"ignoreInboundPorts"`
-	IgnoreOutboundPorts string `json:"ignoreOutboundPorts"`
-	CliVersion          string `json:"cliVersion"`
-	CNIPluginImage      string `json:"cniPluginImage"`
-	CNIPluginVersion    string `json:"cniPluginVersion"`
-	LogLevel            string `json:"logLevel"`
-	PortsToRedirect     string `json:"portsToRedirect"`
-	ProxyUID            int64  `json:"proxyUID"`
-	DestCNINetDir       string `json:"destCNINetDir"`
-	DestCNIBinDir       string `json:"destCNIBinDir"`
-	UseWaitFlag         bool   `json:"useWaitFlag"`
-	PriorityClassName   string `json:"priorityClassName"`
-	ProxyAdminPort      string `json:"proxyAdminPort"`
-	ProxyControlPort    string `json:"proxyControlPort"`
+	InboundProxyPort    uint          `json:"inboundProxyPort"`
+	OutboundProxyPort   uint          `json:"outboundProxyPort"`
+	IgnoreInboundPorts  string        `json:"ignoreInboundPorts"`
+	IgnoreOutboundPorts string        `json:"ignoreOutboundPorts"`
+	CliVersion          string        `json:"cliVersion"`
+	CNIPluginImage      string        `json:"cniPluginImage"`
+	CNIPluginVersion    string        `json:"cniPluginVersion"`
+	LogLevel            string        `json:"logLevel"`
+	PortsToRedirect     string        `json:"portsToRedirect"`
+	ProxyUID            int64         `json:"proxyUID"`
+	DestCNINetDir       string        `json:"destCNINetDir"`
+	DestCNIBinDir       string        `json:"destCNIBinDir"`
+	UseWaitFlag         bool          `json:"useWaitFlag"`
+	PriorityClassName   string        `json:"priorityClassName"`
+	ProxyAdminPort      string        `json:"proxyAdminPort"`
+	ProxyControlPort    string        `json:"proxyControlPort"`
+	Tolerations         []Tolerations `json:"tolerations"`
```

Comment:
As an alternative way of doing this, for `linkerd-control-plane` we don't use a concrete `Toleration` type, and instead use an `[]interface{}` and let the Go compiler figure it all out. 

https://github.com/linkerd/linkerd2/blob/737a4ab7e8f0206871522a0259e61cc36278c549/pkg/charts/linkerd2/values.go#L74

Whatever option we pick, we might want to ensure it's consistent across all the `charts` package?